homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -311,7 +311,7 @@ def preset_mode(self):
     @property
     def available(self):
         """Return true when state is known."""
-        return self._available
+        return super().available and self._available
```

Comment:
The coordinator keeps track of last `last_update_success` and this is used in the coordinator entity to set the `available` property. It doesn't seem needed to also use `self._available` here which is updated when the entity is updated by the coordinator.

Reply:
This is a temporary fix. I'm working on tidying up the fan platform and will remove `available` property from `XiaomiGenericDevice` class. But this needs some time (especially for testing with different devices) and this will be a breaking change (because of removing `mode` and `model` attributes).

Reply:
Ok.